docs/operators/rln-quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quickstart: Running a nwaku node with Rate Limiting Enabled (waku-rln-relay)
+
+This guide explains how to run a nwaku node with RLN (Rate Limiting Nullifier) enabled.
+
+[RLN](https://rfc.vac.dev/spec/32/) is a protocol integrated into waku v2, 
+which prevents spam-based attacks on the network.
+
+For further background on the research for RLN tailored to waku, refer
+to [this](https://rfc.vac.dev/spec/17/) RFC.
+
+Registering to the membership group has been left out for brevity.
+If you would like to register to the membership group and send messages with RLN,
+refer to the [on-chain chat2 tutorial](../tutorial/onchain-rln-relay-chat2.md)
+
+## Prerequisites
+
+1. Follow the [droplet quickstart](./droplet-quickstart.md) or the [build guide](./how-to/build.md) till the `make` command for the wakunode2 binary.
+
+## 1. Update the build command
+
+_Instead_ of just running -
+```bash
+make wakunode2
+```
+
+Run the following command -
+```bash
+make wakunode2 RLN=true
+```
+
+The `RLN=true` flag will enable compilation of the waku-rln-relay protocol
+into your wakunode2 binary.
+
+## 2. Update the runtime arguments
+
+Follow [Step 10](./droplet-quickstart.md#10-run-nwaku) of the [droplet quickstart](./droplet-quickstart.md) guide, while replacing the run command with -
+
+```bash
+export GOERLI_WS_NODE_ADDRESS=<WS RPC URL to a Goerli Node>
+$WAKUNODE_DIR/wakunode2 \
+--store:true \
+--persist-messages \
+--dns-discovery \
+--dns-discovery-url:"$WAKU_FLEET" \
+--discv5-discovery:true \
+--rln-relay:true \
+--rln-relay-dynamic:true \
+--rln-relay-eth-contract-address:0x4252105670fe33d2947e8ead304969849e64f2a6 \
```

Comment:
Could the contract address also be an env variable and then this specific address added as an example? This is our contract (for now), but RLN does not depend on it.

Reply:
Addressed in [31b55aa](https://github.com/status-im/nwaku/pull/1311/commits/31b55aab1df804b2d6b63eb989d3e47939de14d1)